content/en/style-guide/_index.md

```diff
@@ -176,6 +178,90 @@ More about that in the [How To Contribute](/contribute/) doc.
 Before getting started, please read some published Glossary terms 
 to get a feeling for the level of detail and difficulty and when examples are appropriate.
 
+
+
+## Sign Language Video Style Guide {#sign-language-style-guide}
+
+If you are contributing a sign language video, please follow these guidelines to ensure our videos are clear, informative, and accessible to a wide audience (Including hearing community members).
+
+
+### Technical equipment
+
+All you need is a high-resolution camera and a stand. A modern smartphone should be enough.
+
+
+### Recording Setup
+
+
+
+* **Lighting:** Frontal light
+
+
+* **Background:** Record in front of a flat, solid background so it doesn't distract from the signing, and ensure no distracting decorations are included.
+
+
+* **Dress Code:** Solid single-color top (no prints) contrasting your skin tone.
+
+
+* **Camera Position:** Head to elbow should be on the screen and the camera should be positioned near the eye's height. The camera should also be perpendicular to the background (the background behind you should not be slanted).
+
+
+* **Signing Space:** Use as much space as possible, ensuring you have full signing space while sitting or standing.
+
```

Comment:
- **Lighting:** Use frontal lighting to ensure your face and hands are well-lit.

- **Background:** Record in front of a flat, solid background to avoid distractions from the signing. Ensure no distracting decorations are included.

- **Dress Code:** Wear a solid, single-color top (no prints) that contrasts with your skin tone.

- **Camera Position:** Ensure that your head to your elbows are visible on the screen, and position the camera at eye level. The camera should be perpendicular to the background so that it appears straight, not slanted.

- **Signing Space:** Use as much space as possible to ensure you have full signing space while sitting or standing.